tests/testthat/test_formats.R

```diff
@@ -43,10 +43,10 @@ test_format("acm_article")
 test_format("elsevier_article")
 test_format("jss_article")
 test_format("rjournal_article", file_check = FALSE)
-test_format("acs_article", os_skip = "windows")
+test_format("acs_article", file_check = FALSE)
```

Comment:
Tests run fine in mac and Linux. 
However, for windows  I found tests failed and I set the `file_check` to false to pass tests both for acs and ams template. Such templates could be used in windows, just do not pass the unit tests. I think the reason would be the path of the figures generated by the templates.
The pnas template also can not pass the test right now and also could be used in windows. I think `os_skip` would work for it while `file_check` does not work.